### ISSUE TITLE
Written to Material

### ISSUE THREAD

**Issue by rebeccabrover** [holacracyone/Holacracy-Constitution#447]
I'd like to suggest changing 1.2.4 from:
_You are responsible for capturing and tracking all Projects and Next-Actions for your Role in written lists._
to:
_You are responsible for capturing and tracking all Projects and Next-Actions for your Role in **material** lists._

While there are two other uses of _written_ in Relational Agreements and Distributed Authority, I think this particular change further increases sovereignty to capture Projects and Next-Actions using any mechanism a human wishes, while still constraining to somewhere outside of their head to serve the organization. It could be argued that a digital list is not written, or a recording is not written, yet they are both material content that could be made available to others in the absence of the person who crafted them.

*Note - this idea was originally suggested by Holacracy practitioners at Acato Information Management and I have their permission to post it. Shout out to Acato!

**Comment by dienkwik** [comment]
Hi, @rebeccabrover:

Interesting! Do you mind sharing the situation you or the person from Acato was encountering that made it hard to follow this requirement of lists being in written form ?

Thanks.

**Comment by rebeccabrover** [comment]
@dienwik I’m not willing to share the name of the person but I can share the situation. It’s not about it being “hard” to keep a written list, but Acato is a mostly remote tech company and some of the team do not interpret electronic lists as “written” but as another form of “material” content. If the intention is to make sure a role-filler is capturing projects and next-actions somewhere outside of their head, the question to me is whether “written” includes electronic, recording, or other media available to other org members is covered in the current language. By my interpretation, they are not which is why I am suggesting the edit.
I hope that helps.

**Comment by bernardmariechiquet** [comment]
@rebeccabrover This makes total sense to me ! Thanks...
Even if it is not easy to translate!

Could the following works IYO ?

"You are responsible for capturing and tracking all Projects and Next-Actions for your Role in written or numerical lists."
